networks/ssd_inception_v2_gesture/Makefile

```diff
@@ -0,0 +1,166 @@
+	@if [ -e ${GRAPH_FILENAME_BASE_IN_DIR_FP32} ] ; \
+	then \
+		echo "compiled FP32 model file already exists, skipping compile."; \
+	else \
+		echo "Compiling FP32 model..."; \
+		mkdir ssd_inception_v2_mo_fp32 ; \
+		${NCCOMPILE} --data_type=FP32 --reverse_input_channels ${MO_INPUT_GRAPH_ARG} ${MO_INPUT_PIPELINE} ${MO_INPUT_CUSTOM_OPS} ${MO_OUTPUT_DIR_FP32} || (echo $(RED)"Make sure to set the OpenVINO environment variables using the "$(YELLOW)"setupvars.sh"$(RED)" script found in <your OpenVINO install location>/bin/ folder."$(NOCOLOR);  exit 1;); \
+	fi
+	@if [ -e ${GRAPH_FILENAME_BASE_IN_DIR_FP16} ] ; \
+	then \
+		echo "compiled FP16 model file already exists, skipping compile."; \
+	else \
+		echo "Compiling FP16 model..."; \
+		mkdir ssd_inception_v2_mo_fp16 ; \
+		${NCCOMPILE} --data_type=FP16 --reverse_input_channels ${MO_INPUT_GRAPH_ARG} ${MO_INPUT_PIPELINE} ${MO_INPUT_CUSTOM_OPS} ${MO_OUTPUT_DIR_FP16} || (echo $(RED)"Make sure to set the OpenVINO environment variables using the "$(YELLOW)"setupvars.sh"$(RED)" script found in <your OpenVINO install location>/bin/ folder."$(NOCOLOR);  exit 1;); \
+	fi
+	@if [ -e ${MO_LABELS_IN_DIR_FP32} ] ; \
+	then \
+		echo "compiled FP32 model labels already exist, skipping network request."; \
+	else \
+		echo "Grabbing labels for FP32 model..."; \
+		${GET_MO_MODEL_FP32_LABELS}; \
+	fi
+	@if [ -e ${MO_LABELS_IN_DIR_FP16} ] ; \
+	then \
+		echo "compiled FP16 model labels already exist, skipping network request."; \
+	else \
+		echo "Grabbing labels for FP16 model..."; \
+		${GET_MO_MODEL_FP16_LABELS} ;\
+	fi
+
+
+.PHONY: install-reqs
+install-reqs:
+	@echo $(YELLOW)"\nSSD Inception V2 Gesture: Checking application requirements...\n"$(NOCOLOR)
+	@echo "No requirements needed."
+
+
+.PHONY: run_FP16
+run_FP16: install-reqs deps data compile_model
+	@echo $(YELLOW) "\nSSD Inception V2: Running Python sample..." $(NOCOLOR)
+	python3 $(RUN_PY_RELATIVE_DIR) -i 'cam' -m ${GRAPH_FILENAME_BASE_IN_DIR_FP16} --labels ${MO_LABELS_IN_DIR_FP16} -d MYRIAD
+
+
+.PHONY: run_FP32
+run_FP32: install-reqs deps data compile_model
+	@echo $(YELLOW) "\nSSD Inception V2: Running Python sample..." $(NOCOLOR)
+	python3 $(RUN_PY_RELATIVE_DIR) -i 'cam' -m ${GRAPH_FILENAME_BASE_IN_DIR_FP32} -l ${CPU_EXTENSION} --labels ${MO_LABELS_IN_DIR_FP32}
+
+
+.PHONY: run
+run: run_py
+
+
+.PHONY: run_py
+run_py: install-reqs deps data compile_model run_FP16
+
+
+.PHONY: help
+help:
+	@echo "\nPossible make targets"
+	@echo $(YELLOW)"  make help " $(NOCOLOR) "- Shows this message.";
+	@echo $(YELLOW)"  make run " $(NOCOLOR) "- Runs the Python example.";
+	@echo $(YELLOW)"  make run_py " $(NOCOLOR) "- Runs the Python example.";
+	@echo $(YELLOW)"  make all " $(NOCOLOR) "- Makes everything needed to run, but doesn't run the sample.";
+	@echo $(YELLOW)"  make compile_model " $(NOCOLOR) "- Runs model compiler for the network.";
+	@echo $(YELLOW)"  make model " $(NOCOLOR) "- Downloads the model files.";
```

Comment:
This option wasn't implemented, maybe remove from the help?

Also there are other options like run_FP32 which you could add to the help.